jvm/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -209,6 +209,18 @@ case class TransformPreOverrides() extends Rule[SparkPlan] {
           plan.partitionSpec,
           plan.orderSpec,
           replaceWithTransformerPlan(plan.child, isSupportAdaptive))
+      case plan: GlobalLimitExec =>
+        logDebug(s"Columnar Processing for ${plan.getClass} is currently supported.")
+        val child = replaceWithTransformerPlan(plan.child, isSupportAdaptive)
+        LimitTransformer(child, 0L, plan.limit)
+      case plan: LocalLimitExec =>
+        logDebug(s"Columnar Processing for ${plan.getClass} is currently supported.")
+        val child = replaceWithTransformerPlan(plan.child, isSupportAdaptive)
+        LimitTransformer(child, 0L, plan.limit)
+      case plan: CollectLimitExec =>
```

Comment:
there are some different between these three limit operator, only use one `LimitTransformer` to transform them, is it OK?

Add mote UT for there three operators.

Reply:
had added UT for collect limit